pkg/kudoctl/packages/resolver/resolver.go

```diff
@@ -10,32 +12,30 @@ import (
 	"github.com/kudobuilder/kudo/pkg/kudoctl/util/repo"
 )
 
-// Resolver will try to resolve a given package name to either local tarball, folder, remote url or
-// an operator in the remote repository.
-type Resolver interface {
-	Resolve(name string, appVersion string, operatorVersion string) (*packages.Resources, error)
-}
-
 // PackageResolver is the source of resolver of operator packages.
 type PackageResolver struct {
-	local *LocalResolver
-	uri   *URLResolver
+	local *LocalHelper
```

Comment:
Previously, `LocalResolver` and `URLResolver` would implement the `Resolver` interface:
```
Resolve(name string, appVersion string, operatorVersion string) (*packages.Resources, error)
```
However, they are merely helpers to download/parse the directory/tarballs and would ignore the `appVersion` and `operatorVersion` parameters. I kept their overall structure but they don't implement the interface anymore.